qadence/protocols.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import annotations
+
+import importlib
+
+from qadence.logger import get_logger
+
+logger = get_logger(__name__)
+
+try:
+    module = importlib.import_module("qadence_protocols.protocols")
+    available_protocols = getattr(module, "available_protocols")
+except ModuleNotFoundError:
+    raise ModuleNotFoundError(
+        "The 'qadence_protocols' module is not present."
+        " Please install the 'qadence-protocol' package."
+    )
```

Comment:
Does this make the stuff in protocols importable from `qadence` or does the user need to import stuff from `qadence-protocols` ?

Reply:
It doesn't add functionality from `qadence_protocols` to the `qadence` namespace if that's what you mean. Once you've `pip install` protocols, you still have to do:

```python
from qadence_protocols.mitigations.protocol import Mitigations
```

for it to work. Happy to revisit this if found impractical.